src/SwedbankPay.Episerver.Checkout/SwedbankPayCheckoutService.cs

```diff
@@ -96,16 +96,16 @@ public PaymentOrder GetPaymentOrder(IOrderGroup orderGroup, PaymentOrderExpand p
             {
                 var market = _marketService.GetMarket(orderGroup.MarketId);
                 var uri = new Uri(swedbankPayCheckoutOrderId, UriKind.Relative);
-                return GetPaymentOrder(uri, market, paymentOrderExpand);
+                return GetPaymentOrder(uri, market, orderGroup.Properties[Constants.Culture]?.ToString(), paymentOrderExpand);
             }
 
             return null;
         }
 
 
-        public virtual PaymentOrder GetPaymentOrder(Uri id, IMarket market, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
+        public virtual PaymentOrder GetPaymentOrder(Uri id, IMarket market, string languageId, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5375397)